packages/react-components/src/components/gauge/converters/convertSeries.ts

```diff
@@ -82,13 +71,10 @@ export const convertSeries = ({
       formatter: !hasThresholds ? getFormatterValue : '{value}',
       rich: {
         value: {
-          fontWeight: 'bolder', // font weight of the value
-          fontSize: settings?.fontSize,
+          fontSize: 0,
```

Comment:
cant fully remove this value because echarts uses it to draw the gauge, so just hiding the text so we can manually write it